test-network-function/diagnostic/suite.go

```diff
@@ -199,6 +215,18 @@ func listNodeCniPlugins(nodeName string) []CniPlugin {
 	return result
 }
 
+func testOcpVersion() {
+	context := common.GetContext()
+	tester := versionocp.NewVersionOCP(defaultTestTimeout)
+	test, err := tnf.NewTest(context.GetExpecter(), tester, []reel.Handler{tester}, context.GetErrorChannel())
+	gomega.Expect(err).To(gomega.BeNil())
+	testResult, err := test.Run()
+	gomega.Expect(testResult).To(gomega.Equal(tnf.SUCCESS))
+	gomega.Expect(err).To(gomega.BeNil())
+	versionsOcp = tester.GetVersions()
```

Comment:
I think it's better if we have more explicit GetVersions return instead of []strings

Reply:
done